### DISCUSSION TITLE
account was suspended without any warning

### DISCUSSION THREAD

**Discussion by AsangaXd** [community/community#121800] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My main GitHub account was suspended without any prior notification or explanation. As a result, I am unable to access my repositories, projects, and other functionalities associated with my account.

already i open ticket (#2654766) 3 months ago but still haven't any response idk why!!!! 

I kindly request your assistance in investigating the suspension of my account and providing clarification on the reason behind it. Furthermore, I would appreciate any guidance or steps I can take to resolve this issue and regain access to my account.

Thank you for your attention to this matter.

Sincerely,
Asanga Udara

**Comment by satoshibamotonaka** [comment]
same here. NO REASON :(

**Comment by WaddlesADuck** [comment]
Try Creating a New Ticket And I Think The Reason That Happend Beacuse Of One Of These Reasons:
`Violating Terms of Service, Community Guidelines, or Engaging in Malicious Activities such as Spamming, Intellectual Property Infringement, or Fraud.`

**Comment by ebndev** [comment]
Hi @AsangaXd,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
